fetch-in-react-practice/client/src/components/Todos.js

```diff
@@ -8,6 +8,7 @@ export default function Todos() {
   const [todos, setTodos] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const [error, setError] = useState();
+  const [showSpinner, setShowSpinner] = useState(false);
```

Comment:
I'd probably call this "saving" instead of "showSpinner". The fact that it's showing a spinner is an implementation detail in the TodoForm. As far as Todos is concerned, it's just saving to the database.